engines/bastion_katello/app/assets/javascripts/bastion_katello/organizations/organization.factory.js

```diff
@@ -14,7 +14,20 @@ angular.module('Bastion.organizations').factory('Organization',
         return BastionResource('/katello/api/v2/organizations/:id/:action',
             {id: '@id'},
             {
-                update: { method: 'PUT'},
+                get: {
+                    method: 'GET',
+                    transformResponse: function (data) {
+                        // Some fields cannot be PUT back
+                        var org;
+
+                        org = angular.fromJson(data);
+                        delete org.users;
+                        delete org.parent_id;
+                        delete org.parent_name;
+                        return org;
```

Comment:
In the controller all it would be would be something like changing the line here:
https://github.com/Katello/katello/blob/master/engines/bastion_katello/app/assets/javascripts/bastion_katello/subscriptions/manifest/manifest-import.controller.js#L152

from 
```
organization.$update(function (response) {
```

to 

```
Organization.update({'id': organization.id, 'redhat_cdn_url': organization.redhat_cdn_url}, function() {
```

It may be slightly more complex than that, but not much